### DISCUSSION TITLE
Infos about lrclib

### DISCUSSION THREAD

**Discussion by xeruf** [tranxuanthang/lrcget#31] [General]
Hey, great project!

I stumbled upon you via https://www.reddit.com/r/musichoarder/comments/yuwwt9/lrcget_opensource_utility_for_massdownloading_lrc/ and would like to find out more about lrclib and its sources, for example I cannot find the lyrics for this song on your site: https://genius.com/Vinai-and-harris-and-ford-break-the-beat-lyrics

**Comment by tranxuanthang** [comment]
I'm sorry for not being able to reply to you sooner. I don't prefer to talk a lot about LRCLIB and its sources here. The database is currently small (~2M undeduplicated lyrics), but if you can't find your favorite song's lyrics on LRCLIB, you can contribute to it with LRCGET app.

**Reply by koehntopp** [reply]
Let's keep it this way - I have never found a better source for LRC, I don't care where you get it from 🤣

**Reply by horseDeveloper** [reply]
Does a standard for writing lyrics exist?
My question would be: What is the best practice for dividing verses/chorus? An empty line with no timestamp or with a timestamp after the previous line ends? Some of my lyrics are labeled by artist, like "[Verse 1: Flowdan]"

Testing with MusicBee, lines like "[Verse 1: Flowdan]" are not displayed

Edit:

- How are duplicates treated? What happens if I notice I made a mistake and upload an updated version of the synced lyrics?

**Comment by ClaireCJS** [comment]
You could ask people to submit their personal LRCs to you ?  I have a couple 1000

I even have 50,000 .KAS timed-lyric files, but unfortunately those are timing only, and only have the first 2 characters of the text, which must be matched externally.
